src/shared/components/query/StudyListLogic.ts

```diff
@@ -227,6 +227,12 @@ export class FilteredCancerTreeView
 		}
 	}
 
+	@action clearAllSelection(): void
+	{
+        this.store.selectedStudyIds = []
```

Comment:
while this doesn't bother me too much, I do feel like this would be something that might be more natural as a method of the store.

Reply:
i'm just kind of following Andy's separation of concerns, which is pretty different from what we've done.